Mathlib/Computability/TuringMachine.lean

```diff
@@ -2323,9 +2314,10 @@ theorem tr_respects_aux₂ [DecidableEq K] {k : K} {q : Stmt₂₁} {v : σ} {S
       split_ifs with h
         <;> simp only [List.reverse_cons, Function.update_same, ListBlank.nth_mk, List.map]
       -- Porting note: `le_refl` is required.
```

Comment:
Since we're here, is this still required? If not, let's remove it and this porting note.